administrator/language/de-DE/lib_joomla.ini

```diff
@@ -355,6 +355,9 @@ JLIB_FORM_FIELD_PARAM_INTEGER_LAST_DESC="Dieser Wert ist der höchste / letzte i
 JLIB_FORM_FIELD_PARAM_INTEGER_LAST_LABEL="Letzte"
 JLIB_FORM_FIELD_PARAM_INTEGER_STEP_DESC="Jede Option ist die vorherige Option, die um diese ganze Zahl inkrementiert wird, beginnend mit dem ersten Wert, bis der letzte Wert erreicht ist."
 JLIB_FORM_FIELD_PARAM_INTEGER_STEP_LABEL="Schrittweite"
+JLIB_FORM_FIELD_PARAM_LAYOUT_BASIC_SELECT="HTML-Auswahl"
+JLIB_FORM_FIELD_PARAM_LAYOUT_FANCY_SELECT="Erweitert-Auswahl"
```

Comment:
![image](https://github.com/joomlagerman/joomla/assets/66922325/555564c5-340c-4d89-a331-a287c2fe7d5f)
Da müsste glaub noch ein `e` fehlen: Erweitert**e**

Als Alternative wäre auch noch [verbessern oder optimieren](https://www.linguee.de/englisch-deutsch/uebersetzung/enhance.html) (=to enhance) möglich.
`Verbesserte Auswahl` bzw. `Optimierte Auswahl`